addons/mail/wizard/mail_compose_message.py

```diff
@@ -168,7 +168,9 @@ def default_get(self, fields_list):
         compute='_compute_partner_ids', readonly=False, store=True)
     notified_bcc = fields.Many2many(
         string='Bcc', comodel_name='res.partner', compute='_compute_notified_bcc', readonly=True, store=False)
-    show_notified_bcc = fields.Boolean('Show BCC', store=False)
+    notified_bcc_contains_share = fields.Boolean('Is an external partner follower of the document?', compute="_compute_notified_bcc")
+    show_notified_bcc = fields.Boolean('Show BCC', store=False) # TODO: remove field in master
+    all_partners_have_email = fields.Boolean(compute="_compute_all_partners_have_email")
```

Comment:
Maybe partner_ids_all_have_email (to match base m2m name) ? And we define it below 'partner_ids' so that linked fields definitions are put together ?